bedrock/mozorg/templates/mozorg/advertising/landing.html

```diff
@@ -0,0 +1,234 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "base-protocol-mozilla.html" %}
+
+{% from "macros-protocol.html" import callout, card, split with context %}
+
+{% block page_title %}Mozilla Advertising{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-callout') }}
+  {{ css_bundle('protocol-card') }}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('mozilla-ads') }}
+{% endblock %}
+
+{% block sub_navigation %}
+  {% include 'mozorg/advertising/includes/sub-nav.html' %}
+{% endblock %}
+
+{% block content %}
+
+{% call callout(
+  class='mozads-header',
+  title='Mozilla Advertising',
+  desc='Powering the evolution of performance and privacy',
+  heading_level=1,
+  brand=False,
+  content_width='xl',
+  include_cta=True
+) %}
+<p>
+  <a href="https://mozilla.formstack.com/forms/advertising" rel="external noopener" target="_blank" class="mzp-c-button">
+    Contact
+    <span class="mzp-c-button-icon-end">
+      <svg width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+        <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round">
+            <g transform="translate(4.000000, 4.000000)" stroke="currentColor" stroke-width="2">
+                <polyline points="9 0 16 0 16 7"></polyline>
+                <path d="M8,8 L16,0"></path>
+                <path d="M4,-1.77635684e-15 L3,-1.77635684e-15 C1.34314575,-1.47199765e-15 -2.02906125e-16,1.34314575 0,3 L0,13 C2.02906125e-16,14.6568542 1.34314575,16 3,16 L13,16 C14.6568542,16 16,14.6568542 16,13 L16,12"></path>
+            </g>
+        </g>
+      </svg>
+    </span>
+  </a>
+</p>
+{% endcall %}
+
+{% call split(
+  block_class='mozads-t-dark mzp-t-dark mzp-t-content-xl',
+  image=resp_img(
+    url='img/mozorg/advertising/ads-hero.png',
+    srcset={
+      'img/mozorg/advertising/ads-hero-15x.png': '1.5x'
+    },
+    optional_attributes={
+      'loading': 'lazy',
+      'width': '786',
+      'height': '691',
+      'class': 'mzp-c-split-media-asset',
+    },
+  ),
+  body_class='mzp-l-split-v-start',
+  media_class='mzp-l-split-h-center mzp-l-split-v-start',
+) %}
+  <h2>Mozilla is redefining digital advertising with privacy-first, high-performance solutions.</h2>
+
+  <p class="ads-content-intro">Pushing the industry forward with advanced technology,
```

Comment:
Funny story: I was originally just using the `ads-` prefix but when I put the pages on demo I found that my ad blocker was hiding a few parts of the page because of that name, so I changed it to `mozads-`. But clearly I missed some.